Bottom: e62cc5b2ff26ab9f84d6019c39f2277cbb28dcd2
Top:    0257b0ad9f3fb3bbd5ea81c36b06de210c60eadb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:11:30 +0000

Conditional compile different STEP output aspects


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 460719e..f69b538 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -214,20 +214,26 @@ step_do_export (HID_Attr_Val * options)
         poly_DelContour (&curc);
       }
 
+#if 1
     silk_objects = object3d_from_silk_within_area (piece, TOP_SIDE);
     board_outline_list = g_list_concat (board_outline_list, silk_objects);
 
     silk_objects = object3d_from_silk_within_area (piece, BOTTOM_SIDE);
     board_outline_list = g_list_concat (board_outline_list, silk_objects);
+#endif
 
+#if 1
     mask_objects = object3d_from_soldermask_within_area (piece, TOP_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
 
     mask_objects = object3d_from_soldermask_within_area (piece, BOTTOM_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
+#endif
 
+#if 1
     copper_layer_objects = object3d_from_copper_layers_within_area (piece);
     board_outline_list = g_list_concat (board_outline_list, copper_layer_objects);
+#endif
 
   } while ((piece = piece->f) != board_outline);
   poly_Free (&board_outline);
